packages/@o3r/core/schematics/ng-update/v8.2/import-map.ts

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import type { ImportsMapping } from '@o3r/schematics';
+
+/** Map containing the import changes in otter packages for the exported elements */
+export const mapImportAsyncStore: ImportsMapping = {
+  '@o3r/core': {
+    StoreSyncSerializers: {
+      newPackage: '@o3r/async-store'
```

Comment:
It is **@o3r/store-sync** the name of the new package, non ?

Reply:
Well cached !